pkg/kapp/app/recorded_app.go

```diff
@@ -140,17 +157,21 @@ func (a *RecordedApp) mergeAppUpdates(cm *corev1.ConfigMap, labels map[string]st
 }
 
 func (a *RecordedApp) Exists() (bool, string, error) {
-	_, err := a.coreClient.CoreV1().ConfigMaps(a.nsName).Get(context.TODO(), a.name, metav1.GetOptions{})
-	if err != nil {
-		if errors.IsNotFound(err) {
+	_, err := a.coreClient.CoreV1().ConfigMaps(a.nsName).Get(context.TODO(), a.fqName, metav1.GetOptions{})
+	if err == nil {
+		return true, "", nil
+	} else if errors.IsNotFound(err) {
```

Comment:
nit: no need for else, since above return, which would make this code a bit flatter.